src/sagas/wallet.js

```diff
@@ -162,7 +162,10 @@ export function* startWallet(action) {
   // but the wallet may be closed unexpectedly
   const storage = STORE.getStorage();
   yield call([storage.store, storage.store.cleanMetadata]); // clean metadata on memory
-  yield call([storage, storage.cleanStorage], true); // clean transaction history
+  // clean transaction history, addresses, token and nano-contract indexes
+  // XXX: It would be better to keep registered tokens and nano contracts
```

Comment:
Do we have an issue created describing this?

Reply:
Yes, we have:
- #667